standard/ERCs/erc-6900.md

```diff
@@ -100,23 +86,56 @@ Each step is modular, supporting different implementations, that allows for open
 - `IExecutionModule` to support execution functions and their installations on account.
 - `IExecutionHookModule` to support pre & post execution hooks for execution functions.
 
-#### `IModuleManager.sol`
+#### `IModularAccount.sol`
 
-Module manager interface. Modular Smart Contract Accounts **MUST** implement this interface to support installing and uninstalling modules.
+Module execution and management interface. Modular Smart Contract Accounts **MUST** implement this interface to support installing and uninstalling modules, and open-ended execution.
 
 ```solidity
+
 type ModuleEntity is bytes24;
 
 type ValidationConfig is bytes26;
 
 type HookConfig is bytes26;
 
-interface IModuleManager {
+struct Call {
+    // The target address for the account to call.
+    address target;
+    // The value to send with the call.
+    uint256 value;
+    // The calldata for the call.
+    bytes data;
+}
+
+interface IModularAccount {
```

Comment:
Add accountId

Reply:
In case you add this before https://github.com/erc6900/reference-implementation/pull/162 lands, please pull the natspec off of that PR.

Reply:
Added